package Exceptions;


public class Withouttry {

	public static void main(String[] args) {
		int d = 0;
		System.out.println("in the try block");
		int a = 42 / d;
		
		System.out.println("The res of the work in normal course");
		System.out.println("The res of the work in normal course");
		System.out.println("The res of the work in normal course");
		System.out.println("The res of the work in normal course");
		

	}

}
